#! /bin/sh -e

# DP: Fix bashisms in vboxplay script.

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        #cd ${dir}gcc && autoconf
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        #rm ${dir}gcc/configure
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

--- isdnutils-3.9.20060704.orig/vbox/utils/vboxplay.in	2008-04-08 07:45:09.000000000 +0100
+++ isdnutils-3.9.20060704/vbox/utils/vboxplay.in	2008-04-08 07:47:30.000000000 +0100
@@ -15,7 +15,7 @@
 
 SAMPLE=$1
 VOLUME=$2
-if [ "$VOLUME" == "" ]; then
+if [ -z "$VOLUME" ]; then
    VOLUME=5;
 fi
 
